src/vue/common/fees/Fees.vue

```diff
@@ -0,0 +1,250 @@
+            }}
+          </td>
+        </tr>
+      </template>
+
+      <tr class="fees__total-fee-row">
+        <td>
+          {{ 'fees.source-fixed-total-fee' | globalize }}
+        </td>
+        <td v-if="isPayForDestination">
+          {{ getTotalFeeSumByAsset() | formatMoney }}
+        </td>
+        <td v-else>
+          {{
+            getTotalFee({
+              fixed: fees.source.fixed,
+              percent: fees.source.calculatedPercent
+            }) | formatMoney
+          }}
+        </td>
+      </tr>
+
+      <tr
+        class="fees__total-fee-row"
+        v-if="!isWithdrawalFees"
+      >
+        <td>
+          {{ 'fees.destination-fixed-total-fee' | globalize }}
+        </td>
+        <td v-if="isPayForDestination">
+          {{
+            getTotalFee({
+              fixed: 0,
+              percent: 0
+            }) | formatMoney
+          }}
+        </td>
+        <td v-else>
+          {{
+            getTotalFee({
+              fixed: fees.destination.fixed,
+              percent: fees.destination.calculatedPercent
+            }) | formatMoney
+          }}
+        </td>
+      </tr>
+
+      <tr
+        v-if="isAnyFeeForDestination && !isWithdrawalFees"
+      >
+        <td class="fees__tick-field">
+          <tick-field v-model="isPayForDestination">
+            {{ 'fees.pay-fees-for-recipient' | globalize }}
+          </tick-field>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</template>
+<script>
+import { formatMoney } from '@/vue/filters/formatMoney'
+import { MathUtil } from '@/js/utils'
+import IdentityGetterMixin from '@/vue/mixins/identity-getter'
+import FormMixin from '@/vue/mixins/form.mixin'
+import {
+  FEE_TYPES,
+} from '@tokend/js-sdk'
+export default {
+  name: 'fees',
+  components: {},
+  mixins: [
+    FormMixin,
+    IdentityGetterMixin,
+  ],
+  props: {
+    fees: {
+      type: Object,
+      default: () => ({
+        destination: {
+          fixed: 0,
+          calculatedPercent: 0,
+        },
+        source: {
+          fixed: 0,
+          calculatedPercent: 0,
+        },
+      }),
+    },
+    assetCode: {
+      type: String,
+      default: '',
+    },
+    isExternalSystemType: {
+      type: [String, Number],
+      default: 0,
+    },
+  },
+  data () {
+    return {
+      isFeesLoadPending: false,
+      isPayForDestination: false,
+    }
+  },
+  computed: {
+    isWithdrawalFees () {
+      return this.fees.type === FEE_TYPES.withdrawalFee
+    },
+    isAnyFeeForDestination () {
+      return Number(this.fees.destination.fixed) ||
+        Number(this.fees.destination.calculatedPercent)
+    },
+  },
+  watch: {
+    isPayForDestination () {
+      this.$emit('isPayForDestination', this.isPayForDestination)
+    },
+  },
+  methods: {
+    formatMoney,
+    getFeeByAsset (fees) {
+      return {
+        value: fees,
+        currency: this.assetCode,
+      }
+    },
```

Comment:
1. use filters via `this.$filters.something`
2. rename the meothods that were preparing input data for formatMoney filter
3. use the formatMoney from these methods